### PR TITLE
chore: Update index spanish files

### PR DESCRIPTION
**Pull request by electrocucaracha**
The index Spanish files are out of date, they don't have recognitions for contributors, as well as missing some images and new paragraphs. This PR updates the version of those files.

### CODE REVIEW

**Review by krol3 — APPROVED** [review]
LGTM

File: content/es/_index.md [hunk 1]
```diff
@@ -19,6 +21,8 @@ Committee (Business Value Subcommittee) e incluye
...
 [Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) y muchos contribuidores mas. Para una lista de contribuidores completa, consulte [esta página de GitHub](https://github.com/cncf/glossary/graphs/contributors).
 
+Esta versión del glosario Cloud Native en español incluye contribuciones realizadas por [Victor Morales](https://www.linkedin.com/in/electrocucaracha/), [Rael Garcia Arnes](https://www.linkedin.com/in/rael/), [Carol Valencia](https://www.linkedin.com/in/carolgv/), [Anyul Rivas](https://www.linkedin.com/in/anyulled/), [Emiliano Ariel Vazquez](https://www.linkedin.com/in/emiliano-ariel-vazquez-42ba5963/), [Jhan Carlos Silva](https://www.linkedin.com/in/shankyjs/), [Jhonnatan Gil Chaves](https://www.linkedin.com/in/jhonnatan-gil-chaves-57773919/)  y muchos otros colaboradores.
```
Comment (raelga) [inline content/es/_index.md:24]:
```suggestion
Esta versión del glosario Cloud Native en español incluye contribuciones realizadas por [Victor Morales](https://www.linkedin.com/in/electrocucaracha/), [Rael Garcia Arnes](https://www.linkedin.com/in/rael/), [Carol Valencia](https://www.linkedin.com/in/carolgv/), [Anyul Rivas](https://www.linkedin.com/in/anyulled/), [Emiliano Ariel Vazquez](https://www.linkedin.com/in/emiliano-ariel-vazquez-42ba5963/), [Jhan Carlos Silva](https://www.linkedin.com/in/shankyjs/), [Jhonnatan Gil Chaves](https://www.linkedin.com/in/jhonnatan-gil-chaves-57773919/) y muchos otros colaboradores.
```

**Review by raelga — APPROVED** [review]

**Review by raelga — APPROVED** [review]
/lgtm

**Review by seokho-son — APPROVED** [review]
lgtm

**Pull request merged**